app/controllers/api/service_templates_controller.rb

```diff
@@ -20,6 +20,14 @@ def create_resource(_type, _id, data)
 
     def edit_resource(type, id, data)
       catalog_item = resource_search(id, type, collection_class(:service_templates))
+      if data.key?("schedule_time")
+        schedule_time = data&.delete("schedule_time")
```

Comment:
You've already checked the validity of data before this, so the &. is unnecessary.